app/src/main/java/com/asfoundation/wallet/advertise/WalletPoAService.java

```diff
@@ -501,6 +508,12 @@ public void handlePoaCompletedToSendEvent() {
                 .equals(ProofStatus.COMPLETED)) {
               analytics.sendPoaCompletedEvent(proof.getPackageName(), proof.getCampaignId(),
                   Integer.toString(proof.getChainId()));
+            } else if (!proof.getProofStatus()
+                .equals(ProofStatus.PROCESSING) || !proof.getProofStatus()
+                .equals(ProofStatus.SUBMITTING) || !proof.getProofStatus()
+                .equals(ProofStatus.COMPLETED)) {
```

Comment:
Can't this just be an else?
Processing and submitting are not "isTerminate" so if the chain reaches here we know that it wasn't with those 2 cases. And this else is from if(COMPLETED) so no need for that also